README.md

```diff
@@ -2,49 +2,42 @@
 
 _A tool to find ways to get involved on campus._
 
-## Getting Started
+## Contributing
 
-This project requires a working [Node.js](https://nodejs.org/en/) installation
-(at least v14) with NPM, which comes bundled with Node.js (at least v8). (If you
-use Yarn, that's fine, too. Just beware of the commands.)
+We are always open to contributions to the project. If you would like to contribute,
+please join our [Discord](http://discord.utdnebula.com/) and ask for how to access the Jupiter Dev Team
```

Comment:
Does this imply that the project isn't open for public OSS contributions?

Reply:
I think if I add a section about setting up your own environment variables then I can reword this line a bit more. I'll work on it and then resolve this issue once I'm done with that

Reply:
Looks like overall it is improved 👍